docker/types/containers.py

```diff
@@ -417,6 +418,11 @@ def __init__(self, version, binds=None, port_bindings=None,
                 raise host_config_version_error('storage_opt', '1.24')
             self['StorageOpt'] = storage_opt
 
+        if init:
```

Comment:
Minor: this should be `if init is not None:`. If the user explicitly specifies `init=False`, we still want to enter that branch.